boba.gemspec

```
@@ -28,5 +28,5 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 3.0.0"
 
   spec.add_dependency("sorbet-static-and-runtime", "~> 0.5")
-  spec.add_dependency("tapioca", "<= 0.16.7")
+  spec.add_dependency("tapioca", "~> 0.16.9")
```

`<=` was intentional - the way the AR compilers work is that we monkey patch the Tapioca default ones, so changes to Tapioca internals to the compilers can be breaking for Boba but only be a patch version for Tapioca, hence the hard lock. (v.016.10 may introduce an internal refactor, which could be breaking, for instance).

```suggestion
  spec.add_dependency("tapioca", "<= 0.16.9")
```